_data/master/navbars/usage.yml

```diff
@@ -21,6 +21,8 @@ toc:
   path: /usage/configuration/as-service
 - title: Configuring BGP Peers
   path: /usage/configuration/bgp
+- title: Migrating Pools
```

Comment:
Probably want to adjust the sidebar to match the title of the page - "Changing IP Pools"